rasa/core/policies/ted_policy.py

```diff
@@ -919,18 +929,37 @@ def _convert_to_original_shape(
         # mapping the values of attribute features to the position in the resulting
         # tensor.
 
-        batch_dim = tf.size(dialogue_lengths)
-        dialogue_dim = tf.reduce_max(dialogue_lengths)
+        batch_dim = tf.shape(attribute_mask)[0]
+        dialogue_dim = tf.shape(attribute_mask)[1]
         units = attribute_features.shape[-1]
+        if dialogue_lengths is None:
```

Comment:
should we add a comment that we assume that dialogue length is 1 in case it is not provided?